### DISCUSSION TITLE
Copyright Wallet

### DISCUSSION THREAD

**Discussion by colomborkrdz** [MetaMask/snaps#823] [Ideas]
The team I was programmed to work with did not stake their ETH on time and even though they are here in Bogota we won't be able to work on this together.  I'm looking for a team to build on snaps. I come from the music NFT and intellectual property ecosystem. Just published a research paper via the Water and Music research DAO about onchain splits and their applications for music and creators. We identified the need for a tool that allows for multiplayer admin of IP registry (non-transferable) tokens and admin of digital assets. 

The reason why we are interested in building on Meta Mask snaps is to explore pop-up and notification functionality which is where most of the innovation for this design would happen. A step further would require being able to multi-sign off on the deployment of a contract and the minting of the non-transferable token that acts as a registry of intellectual property but from the meta mask snap docs it appears that this is not yet a possibility.

**Comment by Montoya** [comment]
Hi @colomborkrdz, did you find a team to work with?

**Comment by colomborkrdz** [comment]
unfortunately no

**Reply by Montoya** [reply]
Sorry to hear that! If you still plan to build out your idea, let me know! Happy to answer any questions you may have.

**Comment by colomborkrdz** [comment]
Christian thank you. I'm not a developer so I think that the first thing is
to try to onboard somebody interested in building this with me!
